spec/support/features/idv_helper.rb

```diff
@@ -51,6 +51,10 @@ def fill_out_idv_previous_address_fail
     fill_in 'profile_prev_zipcode', with: '00000'
   end
 
+  def fill_out_idv_state_fail
```

Comment:
This method looks to be defined twice

Reply:
One selects the state, the other fills out the state id field.

Reply:
Actually, yes you are right. The names are the same. This was supposed to be `fill_out_idv_state_id_fail`

Reply:
Okay fixed. I just removed the method since it's some functionality I decided was better tested elsewhere.